layouts/partials/footer.hbs

```diff
@@ -58,7 +58,21 @@
     s=o.getElementsByTagName(e)[0];j.async=1;j.src='//www.google-analytics.com/analytics.js';
     s.parentNode.insertBefore(j,s)}(window,document,'ga','script');
 
+    if (!ga) return;
```

Comment:
If you use an ad-blocker, the GA script will not be loaded and thus the `ga` object doesn't exist.